S3/ConnMan.py

```diff
@@ -9,6 +9,7 @@
 import sys
 import httplib
 import ssl
+from _ssl import CERT_NONE
```

Comment:
Good catch.  I hadn't realized it was already available as ssl.CERT_NONE.
Branch updated.

On Mon, Sep 14, 2015 at 3:07 PM, Florent Viard <notifications@github.com>
wrote:

> In S3/ConnMan.py
> <https://github.com/s3tools/s3cmd/pull/621#discussion_r39440135>:
>
> > @@ -9,6 +9,7 @@
> >  import sys
> >  import httplib
> >  import ssl
> > +from _ssl import CERT_NONE
>
> Why not take CERT_NONE from ssl directly?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/s3tools/s3cmd/pull/621/files#r39440135>.
>